src/main/fc/rc.c

```diff
@@ -113,6 +114,11 @@ float getSetpointRate(int axis)
 
 float getRcDeflection(int axis)
 {
+#ifdef USE_ACC
+    if (axis < FD_YAW && (FLIGHT_MODE(ANGLE_MODE) || FLIGHT_MODE(HORIZON_MODE))) {
+        return rcDeflectionSmoothed[axis];
+    }
+#endif
```

Comment:
This can be problematic since the user of `getRcDeflection()` doesn't have control over what value actually gets returned. Additionally, it's not doing what the name suggests. A better way is to split it up into `getRcDeflection()` and `getRcDeflectionSmoothed()`.